docs/source/deploy.rst

```diff
@@ -227,13 +227,11 @@ unix socket:
     After=network.target
 
     [Service]
-    PIDFile=/run/gunicorn/pid
```

Comment:
there's no need for pid files when using systemd, it can track both Type=simple (non-daemonizing like this one) and Type=forking services. 

Additionally it removes the need for the writable directory in `/run`. It's even worse when you have more than one gunicorn service, for example running as different users.